### ISSUE TITLE
possible syntax error?

### ISSUE THREAD

**Issue by martini007-m** [jkminder/nnpatch#1]
hi!
I think in the file site.py line 450, there may be a bug: out[site_name][*index] = site.result.cpu().detach().float(). The * is causing an error in my environment

**Comment by jkminder** [comment]
Hi, you'll need at least python 3.11. I'm sorry i will make that a requirement.

https://peps.python.org/pep-0646/#change-1-star-expressions-in-indexes

**Comment by martini007-m** [comment]
Hi Julian,

Appreciate you letting me know! I have been working on this Machine
learning research paper regarding LLM models and their reliance on context
and prior knowledge. I am wondering if you are one of the authors by any
chance? I have been trying to work through the code from their repository,
and I struggle with the part where you find the layers in a model and
control it in order to manipulate the accuracy.

Anyway, thanks for the help!
Martyna

**Comment by jkminder** [comment]
Do you mean this: https://arxiv.org/abs/2411.07404 ?

We just open sourced the paper. The code is still a bit messy but should be up latest tomorrow. I would love to talk, send me an email at jminder@ethz.ch
